### DISCUSSION TITLE
Automatic Symbolic and Hard Links

### DISCUSSION THREAD

**Discussion by crommy9** [mtkennerly/ludusavi#382] [Ideas]
Not sure how this would work from a permissions standpoint, but with the ability to copy to one-drive or Google Drive. Could you create a hard link, putting a copy of the save folders in one place whilst maintaining updated? The current use case is to keep saves consistently up to date for games without sync between two computers. for instance, PC1:  Game Save Location<<>>One-Drive Folder PC2: One-Drive Folder <<>> Game Save Location.

**Comment by mtkennerly** [comment]
Hi! I considered something like this, but ultimately decided against it. Here was my reasoning: https://github.com/mtkennerly/ludusavi/issues/11#issuecomment-1452618152

**Reply by crommy9** [reply]
Thank you for the quick response, totally makes sense. Shame it isn't as simple as I thought, it would be a great feature keeping items synced accross devices.
